vertical-pod-autoscaler/FAQ.md

```diff
@@ -162,6 +162,16 @@ spec:
 
 and then deploy it manually if your vpa is already configured.
 
+## Can I run the VPA in an HA configuration?
```

Comment:
@adrianmoisey, just want to confirm, that `Can I run the VPA in an HA configuration?` is under the `I get recommendations for my single pod replicaSet but they are not applied` content if yes then it should be 
`#### Can I run the VPA in an HA configuration?` instead of `## Can I run the VPA in an HA configuration?`

If not(which I think) then it should be `### Can I run the VPA in an HA configuration?` and you have to mention this under the [Contents](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/FAQ.md?plain=1#L3)

Sorry forgot to mention previously.

Reply:
Good catch, fixed in cccebf1a41